agent/util-scripts/pbench-tool-meister-client

```diff
@@ -77,11 +77,8 @@ def main(argv: list) -> int:
         logger.error(str(exc))
         return exc.return_code
 
-    with Client(
-        redis_host=redis_server.host,
-        redis_port=redis_server.port,
-        channel_prefix=cli_tm_channel_prefix,
-        logger=logger,
+    with Client.create_with_redis(
```

Comment:
Is there a use of `Client` with out the `create_with_redis`?

Reply:
You can also use `create_with_exporter`. Additionally, you can use the `Client`'s standard `__init__` function, and pass in an existing `SignalExporter` as well as `False` for `to_be_shutdown`.